book/src/csi-driver-object.md

```diff
@@ -36,60 +36,63 @@ There are three important fields:
   * If a `CSIDriver` object does not exist for a given CSI Driver, the default is `true` -- meaning attach will be called.
   * If a `CSIDriver` object exists for a given CSI Driver, but this field is not specified, it also defaults to `true` -- meaning attach will be called.
   * For more information see [Skip Attach](skip-attach.md).
-* `podInfoOnMountVersion`
+* `podInfoOnMount`
   * Indicates this CSI volume driver requires additional pod information (like pod name, pod UID, etc.) during mount operations.
-  * If value is not specified, pod information will not be passed on mount.
-  * If value is set to a valid version, Kubelet will pass pod information as `volume_context` in CSI `NodePublishVolume` calls.
-  * Supported versions:
-    * Version "v1" will pass the following additional fields in `volume_context`:
-	  * `"csi.storage.k8s.io/pod.name": pod.Name`
-	  * `csi.storage.k8s.io/pod.namespace": pod.Namespace`
-	  * `csi.storage.k8s.io/pod.uid": string(pod.UID)`
+  * If value is not specified or `false`, pod information will not be passed on mount.
+  * If value is set to `true`, Kubelet will pass pod information as `volume_context` in CSI `NodePublishVolume` calls:
+    * `"csi.storage.k8s.io/pod.name": pod.Name`
+    * `csi.storage.k8s.io/pod.namespace": pod.Namespace`
+    * `csi.storage.k8s.io/pod.uid": string(pod.UID)`
   * For more information see [Pod Info on Mount](pod-info.md).
 
 ## What creates the CSIDriver object?
 
 CSI drivers do not need to create the `CSIDriver` object directly. Instead they may use the [cluster-driver-registrar](cluster-driver-registrar.md) sidecar container (customizing it as needed with startup parameters) -- when deployed with a CSI driver it automatically creates a `CSIDriver` CR representing the driver.
```

Comment:
Update this section to not mention "CRD or "CR". Suggestion:

CSI drivers do not need to create the `CSIDriver` object directly. Instead they may use the [cluster-driver-registrar](cluster-driver-registrar.md) sidecar container (customizing it as needed with startup parameters). When deployed with a CSI driver, this sidecar will automatically creates a `CSIDriver` object to represent the driver.